src/common/controllers/ads-controller.js

```diff
@@ -293,13 +311,18 @@ class AdsController extends FakeEventTarget implements IAdsController {
   _onAdStarted(event: FakeEvent): void {
     this._ad = event.payload.ad;
     this._adPlayed = true;
+    this._isAdPlaying = true;
   }
 
   _onAdBreakEnd(): void {
     this._adBreak = null;
     this._ad = null;
   }
 
+  _onAdCompleted(): void {
```

Comment:
Why new function for one line?
Just let's be consistent - or add new functions also for `AD_PAUSED` and `AD_RESUMED`, or make it anonymous function for as you did for `AD_PAUSED` and `AD_RESUMED`

Reply:
done